components/datetime/src/skeleton/helpers.rs

```diff
@@ -297,10 +298,9 @@ fn group_fields_by_type(fields: &[Field]) -> FieldsByType {
 fn adjust_pattern_field_lengths(fields: &[Field], pattern: &mut Pattern) {
     runtime::helpers::maybe_replace(pattern, |item| {
         if let PatternItem::Field(pattern_field) = item {
-            if let Some(requested_field) = fields
-                .iter()
-                .find(|field| field.symbol == pattern_field.symbol)
-            {
+            if let Some(requested_field) = fields.iter().find(|field| {
+                field.symbol.discriminant_cmp(&pattern_field.symbol) == Ordering::Equal
```

Comment:
> nipick (style | optional)

Rust 1.53 introduced the [`is_eq()`](https://doc.rust-lang.org/stable/std/cmp/enum.Ordering.html#method.is_eq) function for `Ordering`. 

I'd like to see this be 

```rust
field.symbol.discriminant_cmp(&pattern_field.symbol).is_eq()
```

Reply:
Done